compute-at-edge-test/src/index.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="@fastly/js-compute" />
```

Comment:
Let's create a new top level directory "fastly-compute" and put this in there... perhaps

`fastly-compute/geoip/src/index.js`